src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -14,6 +14,8 @@ public class TorpedoStore {
 
   private int torpedoCount = 0;
 
+  private Random random = new Random();
```

Comment:
Én nem így csináltam, hanem a konstruktorba raktam ezt bele, de elvileg így is működik